src/DayColumn.js

```diff
@@ -77,6 +77,44 @@ class DayColumn extends React.Component {
     this.slotMetrics = this.slotMetrics.update(nextProps)
   }
 
+  renderSelection() {
+    const { selecting } = this.state
+
+    if (selecting) {
+      let { top, height, startDate, endDate, noMovementYet } = this.state
+
+      const { localizer, components } = this.props
+
+      let selectDates = { start: startDate, end: endDate }
+
+      let selectionElement
+      if (
+        noMovementYet &&
+        components &&
+        components.beforeSelectIndicatorComponent
+      ) {
+        // if the user is about to start selecting but hasn't started dragging yet
+        selectionElement = components.beforeSelectIndicatorComponent({
```

Comment:
this should be a real component not a render prop to be consistent with the other apis

Reply:
Roger that, I'll make the change.